### PR TITLE
refactor: remove js import extensions

### PR DESCRIPTION
**Pull request by Trim21**

### CODE REVIEW

File: ui/dps/rdmty/dps.js [hunk 1]
```diff
@@ -42,44 +42,44 @@ var ____Class0=React.Component;for(var ____Class0____Key in ____Class0){if(____C
         return (
             this.props.perSecond === '---' ? null :
             React.createElement("li", {
-                className: 'row ' + this.props.job.toLowerCase() + (this.props.isSelf ? ' self' : ''), 
-                onClick: this.props.onClick}, 
+                className: 'row ' + this.props.job.toLowerCase() + (this.props.isSelf ? ' self' : ''),
+                onClick: this.props.onClick},
```
Comment (MaikoTan) [inline ui/dps/rdmty/dps.js:46]: ？

**Review by MaikoTan — CHANGES_REQUESTED** [review]

File: ui/dps/rdmty/dps.js (hunk 1 above)
Comment (Trim21) [reply ui/dps/rdmty/dps.js:46]: Why there are many trailing whitespaces in this file...

Comment (MaikoTan) [reply ui/dps/rdmty/dps.js:46]: But it is not the point in this PR, recover it.

**Review by MaikoTan — APPROVED** [review]

File: ui/dps/rdmty/dps.js (hunk 1 above)
Comment (quisquous) [reply ui/dps/rdmty/dps.js]: (Thank you!)

**Pull request merged**